src/scss/_tooltips.scss

```diff
@@ -88,9 +87,9 @@
 .tooltip-left:hover::after,
 .tooltip-left:focus::before,
 .tooltip-left:focus::after {
-  -webkit-transform: translateX(-0.8rem);
-  -moz-transform: translateX(-0.8rem);
-  transform: translateX(-0.8rem);
+  -webkit-transform: translateX(-0.6rem);
```

Comment:
we shouldn't have to include vendor prefixes since we use autoprefixes (see https://github.com/go-vela/ui/blob/master/.postcssrc.js)

Reply:
oh good point I knew that, copy pasta. fixed!